resources/build/trigger-builds.sh

```diff
@@ -6,22 +6,24 @@
 
 function triggerBuilds() {
   local base=`git branch --show-current`
-  if [[ $base == master ]]; then
-    local TEAMCITY_BUILDTYPES=( KeymanAndroid_Build Keyman_iOS_Master KeymanLinux_Master KeymanMac_Master Keyman_Build Keymanweb_Build )
-    local TEAMCITY_VCS_ID=HttpsGithubComKeymanappKeyman
-  elif [[ $base == beta ]]; then
-    local TEAMCITY_BUILDTYPES=( KeymanAndroid_Beta Keyman_iOS_Beta KeymanLinux_Beta KeymanMac_Beta KeymanDesktop_Beta Keymanweb_Beta )
-    local TEAMCITY_VCS_ID=Keyman_KeymanappKeymanBeta
-  else
-    exit 1
-  fi
+  eval TEAMCITY_VCS_ID='${'vcs_$base'}'
+  echo "base=$base, TEAMCITY_VCS_ID=${TEAMCITY_VCS_ID}"
 
-  for TEAMCITY_BUILDTYPE in "${TEAMCITY_BUILDTYPES[@]}"; do
-    triggerBuild $TEAMCITY_BUILDTYPE $TEAMCITY_VCS_ID
+  for platform in ${available_platforms[@]}; do
+    eval builds='(${'bc_${base}_${platform}'[@]})'
+    for build in "${builds[@]}"; do
+      if [[ $build == "" ]]; then continue; fi
+      if [ "${build:(-8)}" == "_Jenkins" ]; thend
```

Comment:
```suggestion
      if [ "${build:(-8)}" == "_Jenkins" ]; then
```

Reply:
What!? How did that happen? I tested it before committing! :smiley:

Reply:
It's also quite common to see `:wq!` in scripts. 🤣